embedded-graphics/CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 - **(breaking)** [#312](https://github.com/jamwaffles/embedded-graphics/pull/312) The methods in the `Dimension` trait are replaced by a single `bounding_box` method that returns a `Rectangle`.
 - **(breaking)** [#353](https://github.com/jamwaffles/embedded-graphics/pull/353) Lines with equal start and end points are now drawn by assuming they are oriented horizontally. This means that they are now drawn as a `stroke_width` high and 1px wide rectangle, instead of not being drawn at all.
 - **(breaking)** [#353](https://github.com/jamwaffles/embedded-graphics/pull/353) `primitives::line::StyledLineIterator` was renamed to `primitives::line::StyledIterator`.
+- **(breaking)** [#342](https://github.com/jamwaffles/embedded-graphics/pull/342) Refactored the `DrawTarget` trait to better support common hardware capabilities.
```

Comment:
This will eventually need a migration guide item describing how to migrate in detail.

Reply:
Added :+1: